release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+- [cht-core#6410](https://github.com/medic/cht-core/issues/6410): Ability to customize icons shown on main tabs (contact, tasks, etc)
+- [cht-core#6419](https://github.com/medic/cht-core/issues/6419): Allow you to configure outbound to send the same record multiple times
+- [cht-core#6481](https://github.com/medic/cht-core/issues/6481): Allow enough space to show list of forms in Reports Filter. 
+- [cht-core#6492](https://github.com/medic/cht-core/issues/6492): Touch area of items in the hamburger menu should be larger
+- [cht-core#6516](https://github.com/medic/cht-core/issues/6516): Improve SMS reminders so they can be configured to be sent to places anywhere in the hierarchy
+- [cht-core#6538](https://github.com/medic/cht-core/issues/6538): Adhere to play store privacy policy requirements update 
+- [medic-conf#273](https://github.com/medic/medic-conf/issues/273): Add configurable hierarchy support to doc references in csv-to-docs
+
+### Performance fixes
+
+- [cht-core#6116](https://github.com/medic/cht-core/issues/6116): Improve the performance of deleting read docs
+- [cht-core#6276](https://github.com/medic/cht-core/issues/6276): Use http2 to fetch all login resources at the same time
+
+### Bug fixes
+
+- [cht-core#6024](https://github.com/medic/cht-core/issues/6024): The outbound error response logging is needlessly verbose
+- [cht-core#6121](https://github.com/medic/cht-core/issues/6121): Translations in the App Settings app aren't recognized until you refresh
+- [cht-core#6130](https://github.com/medic/cht-core/issues/6130): Reports deleted from webapp interface are saved fully hydrated
+- [cht-core#6182](https://github.com/medic/cht-core/issues/6182): Support custom contact types in medic-pyxform
+- [cht-core#6313](https://github.com/medic/cht-core/issues/6313): Use the browser language as the default language
+- [cht-core#6370](https://github.com/medic/cht-core/issues/6370): Replication "isSensitive" logic prevents "boss" from creating any report for users directly under them in hierarchy
+- [cht-core#6386](https://github.com/medic/cht-core/issues/6386): Error on report not cleared
+- [cht-core#6404](https://github.com/medic/cht-core/issues/6404): User name autofilled with "medic" 
+- [cht-core#6433](https://github.com/medic/cht-core/issues/6433): Infinite spinners when loading tabs as admin
+- [cht-core#6502](https://github.com/medic/cht-core/issues/6502): Guided tour erratic when user has no access to any tab
+- [cht-core#6519](https://github.com/medic/cht-core/issues/6519): Upgrading doesn't apply replication configuration which means `users-meta` is empty
+- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Muso production users have multiple task documents in non-terminal state with the same emission id
+- [cht-core#6567](https://github.com/medic/cht-core/issues/6567): Security issue medic-infrastructure#147
+- [cht-core#6568](https://github.com/medic/cht-core/issues/6568): Security issue medic-infrastructure#190
```

Comment:
I think we should list security improvements separately... maybe we need a new Type label?

Anyway let's disclose these a little better - right now nobody outside of the org can see what we did. Think about what we can say without disclosing too much about the actual vulnerability?